packages/core/getBusyTimes.ts

```diff
@@ -343,15 +343,17 @@ export async function getBusyTimesForLimitChecks(params: {
       },
       title: true,
       userId: true,
+      attendees: true,
```

Comment:
I tried _count: { select: { attendees: true } } and indeed that doesn't work.

I think this should be: attendees: { select: { id: true } } to ensure minimal perf hits; also; ideally we only run the attendees subQuery when it is a seatsPerTimeSlot > 1 situation.

What do you think?

Reply:
> I think this should be: attendees: { select: { id: true } } to ensure minimal perf hits.

Thanks for the suggestion. I have implemented it.

As for the subQuery; yeah it is necessary only for the seated events.

I have started working on this PR after quite some time and would like to stabilize it a bit. 
After that I can look into the conditional subQuery.